processed 5 tasks

task 1 'publish'. lines 8-24:
created: object(103)
written: object(102)

task 2 'run'. lines 26-26:
Error: Error executing ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: InvalidNonEntryFunction, source: None } }

task 3 'run'. lines 28-28:
Error: Error executing ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: InvalidNonEntryFunction, source: None } }

task 4 'run'. lines 30-30:
Error: Error executing ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: InvalidNonEntryFunction, source: None } }
